app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx

```diff
@@ -58,24 +64,68 @@ export class DashboardItemPage extends Component {
   static propTypes = {
     intl: intlShape.isRequired,
     showModalAction: PropTypes.func.isRequired,
-    dashboardId: PropTypes.string,
+    dashboardId: PropTypes.string.isRequired,
+    activeProject: PropTypes.string.isRequired,
+    dashboardItems: PropTypes.array,
   };
 
   static defaultProps = {
-    dashboardId: undefined,
+    dashboardItems: [],
   };
 
   state = {
     isFullscreen: false,
+    dashboardName: '',
```

Comment:
It's not needed to keep dashboardName in local state.
You have all access to dashboard items received with props. 
You can get active dashboard name just filtering dashboardItems array by id with dashboardId prop.